odoo/tests/common.py

```diff
@@ -1672,6 +1672,16 @@ def request(self, *args, **kwargs):
         return super().request(*args, **kwargs)
 
 
+class CrossModule():
+
+    registry = OrderedSet()
```

Comment:
First, `registry` is not a good idea, since `TransactionCase` uses that attribute for something else!
Second, you don't need an `OrderedSet`, since a subclass will never register twice and you never test `x in CrossModule.registry`.  A simple list will do the job just fine :wink:

Reply:
I don't remember why I used an OrderedSet here. I'll remove it since I can't justify why.